api/api.go

```diff
@@ -169,8 +169,10 @@ func (a *API) ListenExecution(service string, f *ExecutionFilter) (*ExecutionLis
 			if err != nil {
 				return nil, err
 			}
-			if _, err := task.GetOutput(f.OutputKey); f.OutputKey != "" && err != nil {
-				return nil, err
+			if f.HasOutputKey() {
+				if _, err := task.GetOutput(f.OutputKey); f.OutputKey != "" && err != nil {
```

Comment:
`f.OutputKey != ""` Is this still necessary?

Reply:
What do you mean?

The output must exist in the filter in order to get output and check if exists. If the output is empty or `*` then it doesn't check. So basically this line is necessary or am I missing your question

Reply:
The `f.OutputKey != ""` check is in function `HasOutputKey` so it doesn't need to be in the condition line 176.
But anyway, this is not blocking. So let's leave it.

Reply:
fixed, wating for tests to complete